packages/react-components/react-jsx-runtime/src/utils/DevRuntime.native.ts

```diff
@@ -0,0 +1,7 @@
+import { createReactStrictDomJSX } from '@fluentui/react-platform-adapter';
+import * as ReactDevRuntime from 'react/jsx-dev-runtime';
+import type { JSXRuntime } from './types';
+
+export const DevRuntime = {
+  jsxDEV: createReactStrictDomJSX((ReactDevRuntime as { jsxDEV: JSXRuntime }).jsxDEV),
```

Comment:
Rather than forking the DevRuntime and Runtime files can we just have a function in the platform adapter that returns JSXRuntime in the web implementation and does the createReactStrictDomJSX call in the native implementation? It feels dangerous to have *.native.ts files in the main fluent projects.